3001-4000/LIT3200DossierEw.xml

```diff
@@ -29,48 +30,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
       </fileDesc>
       <encodingDesc>
-         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
             <xi:fallback>
                <p>Definitions of prefixes used.</p>
             </xi:fallback>
          </xi:include>
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <ab>The collection consisiting Life (in one of several redactions), Miracles (varying number),
-               and Hymns in honour of St ʾEwosṭātewos</ab>
+            <ab>The collection consisits of the Life (in one of several redactions), Miracles (in a varying
```

Comment:
in the general record, "recension" has been used. I think it would be good to use the same term in all records.